src/storage/txn/commands/resolve_lock.rs

```diff
@@ -93,7 +95,20 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for ResolveLock {
                     false,
                 )?
             } else if commit_ts > current_lock.ts {
-                commit(&mut txn, &mut reader, current_key.clone(), commit_ts)?
+                // Continue to resolve locks if the not found committed locks are pessimistic type.
+                // They could be left if the transaction is finally committed and pessimistic conflict
+                // retry happens during execution.
+                match commit(&mut txn, &mut reader, current_key.clone(), commit_ts) {
+                    Ok(res) => res,
+                    Err(MvccError(box MvccErrorInner::TxnLockNotFound { .. }))
+                        if current_lock.is_pessimistic_lock_type() =>
+                    {
+                        info!("skip resolve commit pessimistic lock error as it's not found";
```

Comment:
I don't think this log is necessary since we already log once in commit, and the debug fmt of `Lock` is very verbose.😂